dialogy/plugins/text/list_entity_plugin/__init__.py

```diff
@@ -85,16 +90,39 @@ def __init__(
             const.REGEX: self.regex_search,
         }
 
-        self.style = style or const.REGEX
+        self.style = style or const.REGEX # which search algo will be used: [regex, spacy, fuzzy]
         self.labels = labels
         self.keywords = None
         self.spacy_nlp = spacy_nlp
         self.compiled_patterns: Dict[str, Dict[str, List[Any]]] = {}
         self.flags = flags
+        '''
+        Parameters for Fuzzy Dependency Parser defined below
+        '''
+        self.file_path = search_config_path
+        self.entity_dict = {}
+        self.entities = {}
+        self.nlp = {}
 
         if self.style == const.REGEX:
             self._parse(candidates)
 
+        if self.style == 'fuzzy_dp_search':
+            self.fuzzy_init()
+
+    def fuzzy_init(self):
+        """
+        Initializing the parameters for fuzzy dp search with their values 
+
+        """
+        with open(self.file_path) as entity_file:
+            parsed_file =  yaml.load(entity_file, Loader=yaml.SafeLoader)
+        for i in parsed_file.keys():
```

Comment:
What is `i`? if the keys are language codes, can we use `lang_code` or something that specifies it?